source/administration/config-settings.rst

```diff
@@ -1504,7 +1504,7 @@ Require Email Verification
 
 Email Invite Salt
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-32-character (to be randomly generated via System Console) salt added to signing of email invitation links. Email invitation links expire after 24 hours except if the salt is regenerated, then existing email invitation links invalidate immediately.  Click **Regenerate** to create new salt.
+32-character (to be randomly generated via System Console) salt added to signing of email invitation links. Click **Regenerate** to create new salt.
```

Comment:
Can we keep the text as is (unless it's inaccurate?), but just change the 24 > 48?

Reply:
The thing is exactly that, is no longer true. Some time ago we separate entirely the concept of InvitationId and the invitation by email. So, now, if you change the invitation id, it doesn't revoke the current existing invitations for the team. I going to check it manually, but I'm almost sure.